marked-element.html

```diff
@@ -285,6 +285,34 @@
       this.fire('marked-render-complete', {}, {composed: true});
     },
 
+     /**
+     * Fired when the content is being processed and before it is rendered.
+     * Provides an opportunity to implement syntax highlighting, [e.g with highlightjs](https://highlightjs.org/).
```

Comment:
I agree with @stramel in that this is not the best place to put this example. I recommend moving the example up top so that it doesn't crowd the [`methods`](https://www.webcomponents.org/element/PolymerElements/marked-element/elements/marked-element#methods) section of the webcomponents.org docs